### ISSUE TITLE
Remove window.location references so lib can be compatible with SSR

### ISSUE THREAD

**Issue by JoshuaNitschke** [luukhaijes/kinde-angular#20]
**Is your feature request related to a problem? Please describe.**
I tried to make a brand new project with SSR enabled, and provideKinde() errors due to references to the window api.

**Describe the solution you'd like**
Is there a reason for the current implementation vs using the native Location API from Angular?
import { Location } from '@angular/common';

**Describe alternatives you've considered**
I suppose that I could use the kinde sdk directly instead of this convenience wrapper, but the wrapper is nice!

**Additional context**
Here is the full error:
```
3:39:36 PM [vite] Error when evaluating SSR module /main.server.mjs:
|- ReferenceError: window is not defined
    at Module.provideKinde (D:\GIT\********\.angular\cache\19.0.2\Zabaan\vite\deps_ssr\kinde-angular.js:14866:15)
```

What do you think?

**Comment by JoshuaNitschke** [comment]
I seem to have auth flow working now on my SSR app if I install from a local NPM package build from my PR: 
https://github.com/luukhaijes/kinde-angular/pull/21

**Comment by luukhaijes** [comment]
Hey @JoshuaNitschke, there is no particular reason why I used the current implementation for `location`. I like your proposed solution in #21. Code looks good and I will test it later, thanks for your work on this library, much appriciated.

**Comment by JoshuaNitschke** [comment]
Many thanks, @luukhaijes !

**Comment by luukhaijes** [comment]
Fixed by #21
